### PR TITLE
Remove no longer used plonetheme.teamraum uninstall profile

### LINKED ISSUE #4406: Remove plonetheme.teamraum GenericSetup uninstall tests
Now that we've integrated it we can drop the uninstallability tests.

### PR DESCRIPTION
**Pull request by elioschmutz**
closes #4406 

Remove no longer used plonetheme.teamraum uninstall profile since it's integrated into gever.

### CODE REVIEW

File: plonetheme/teamraum/tests/test_uninstall.py [hunk 1]
```diff
@@ -1,54 +0,0 @@
-from ftw.testing.genericsetup import apply_generic_setup_layer
-from ftw.testing.genericsetup import GenericSetupUninstallMixin
-from unittest2 import TestCase
-
-
-@apply_generic_setup_layer
```
Comment (Rotonen) [inline plonetheme/teamraum/tests/test_uninstall.py]: This should also get removed from the layer whitelist of the new testrunner setup if it is the last of its kind:

https://github.com/4teamwork/opengever.core/blob/7ed3dbf6aa29eea4f28d8bbb3513726332260e04/test-plone-4.3.x.cfg#L15

Comment (elioschmutz) [reply plonetheme/teamraum/tests/test_uninstall.py]: 👍

**Comment by elioschmutz** [comment]
@Rotonen

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -5,6 +5,7 @@ Changelog
 2018.4.0 (unreleased)
 ---------------------
 
+- Remove no longer used plonetheme.teamraum uninstall profile since it's integrated into plone. [elioschmutz]
```
Comment (phgross) [inline docs/HISTORY.txt]: into `opengever.core` not `plone`

**Comment by elioschmutz** [comment]
@phgross @deiferni 👍

**Pull request merged**